object_detection/setup.md

```diff
@@ -42,6 +42,10 @@ Let's make use of the app to continue with the tutorial.
 
 ```
 cd ks-app
+ks env rm default
+ENV=default
+ks env add ${ENV} --context=`kubectl config current-context`
```

Comment:
Why are you explicitly setting context to the current-context? Won't ks do this automatically?

Reply:
ks do this only when run "ks init" when create ks_app.

Reply:
yes, agree with you. We don't need create an existing env. Updated.